docs/hpc/09_ood/04_RStudio.mdx

```diff
@@ -0,0 +1,37 @@
+# RStudio in OOD
+
+```
```

Comment:
This page has been accidentally commented out.

Reply:
same as above